src/components/Carousel/components/Buttons/Buttons.js

```diff
@@ -19,76 +19,47 @@ import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 import { ChevronLeft, ChevronRight, Pause, Play } from '@sumup/icons';
 
-import CircuitUIButton from '../../../Button';
+import IconButton from '../../../IconButton';
 
 const buttonListStyles = css`
   label: carousel__buttonlist;
   display: flex;
   align-items: center;
   justify-content: center;
 `;
+
 export const ButtonList = styled('div')(buttonListStyles);
 
-const baseButtonStyles = ({ theme }) => css`
+const buttonStyles = ({ theme }) => css`
   label: carousel__button;
-  background-color: ${theme.colors.n300};
-  width: ${theme.iconSizes.giga};
-  height: ${theme.iconSizes.giga};
-  padding: 0;
-  overflow: hidden;
-  border-radius: 50%;
-
   margin-left: ${theme.spacings.byte};
 
   &:first-of-type {
     margin-left: 0;
   }
-  &:active,
-  &:focus,
-  &:hover {
-    padding: 0;
-  }
-
-  ${theme.mq.untilKilo} {
-    width: ${theme.iconSizes.mega};
-    height: ${theme.iconSizes.mega};
-
-    svg {
-      width: 25%;
-    }
-  }
 `;
-export const Button = styled(CircuitUIButton)(baseButtonStyles);
 
-// wrapping button content due to flex bug in Safari 10.1
```

Comment:
Was this issue addressed?

Reply:
Yup! The IconButton never had this issue, one of the reasons why I replaced the custom button here.

You can verify it yourself thanks to the [deploy preview](https://circuit-ui-git-feature-polish-button.sumup-oss.now.sh/?path=/story/components-carousel--stateful) 😉

Reply:
While checking this, I found another issue related to the resized icons though 👀

Reply:
Fixed in 771b821.